src/plugins/discover/public/application/components/default_discover_table/default_discover_table.tsx

```diff
@@ -87,15 +91,13 @@ export const LegacyDiscoverTable = ({
         <table data-test-subj="docTable" className="osd-table table">
           <thead>
             <TableHeader
-              displayedTableColumns={displayedTableColumns}
-              defaultSortOrder={''}
-              // hideTimeColumn,
+              displayedColumns={displayedDefaultColumns}
+              defaultSortOrder={defaultSortOrder}
               indexPattern={indexPattern}
-              // isShortDots,
-              onChangeSortOrder={onChangeSortOrder}
+              onChangeSortOrder={onSort}
               onReorderColumn={onReorderColumn}
               onRemoveColumn={onRemoveColumn}
-              sortOrder={sortOrder}
+              sortOrder={sort as SortOrder[]}
```

Comment:
is this needed? sort is already typed correctly above right?

Reply:
will remove this type